src/patches/dense_patch_computation.cpp

```diff
@@ -446,8 +427,8 @@ InstructionApplicationResult try_apply_instruction_direct_followup(
                         });
 
                 if(could_not_find_space_for_patch)
-                    return {std::make_unique<std::runtime_error>(
-                            "Could not find space to place patch after BusyRegion clears"),{std::move(instruction)}};
+                    return {std::make_unique<std::runtime_error>(lstk::cat(
+                            instruction,"; Could not find space to place patch after BusyRegion clears")),{}};
```

Comment:
Now there is no more instance of both an error and follow up instructions. This data-structure should really be refactored into a variant. I assume you have uncommited changes though, so leaving this change for another time to reduce merge conflicts on your end.